docker/jenkins/common/install_anaconda.sh

```diff
@@ -26,22 +27,14 @@ case "$ANACONDA_VERSION" in
   ;;
 esac
 /bin/bash ~/anaconda.sh -b -p /opt/conda
-rm ~/anaconda.sh
+rm -f ~/anaconda.sh
```

Comment:
Why?

Reply:
I thought it might be the cause of another problem. It wasn't, but it didn't seem like a bad thing to do anyway

Reply:
It's not needed though